index.html

```diff
@@ -49,26 +75,25 @@ <h2>Our Mission</h2>
   
   <p>Recipients should be able to easily access the content of every email sent to them, regardless of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation and sending tools, and email clients to drive this vision forward at every step of the email journey.</p>
   
-  <h2>Quick Links</h2>
+  <h2>Quick Links</h2><img class="new-tab" src="img/new-tab-icon.png" alt="new tab icon">
```

Comment:
Do we have `img/new-tab-icon.png`?

Reply:
img folder and icon images have now been added